src/Akka.Hosting.API.Tests/verify/CoreApiSpec.ApproveRemoting.verified.txt

```diff
@@ -6,13 +6,66 @@
         public static Akka.Hosting.AkkaConfigurationBuilder WithRemoting(this Akka.Hosting.AkkaConfigurationBuilder builder, System.Action<Akka.Remote.Hosting.RemoteOptions> configure) { }
         public static Akka.Hosting.AkkaConfigurationBuilder WithRemoting(this Akka.Hosting.AkkaConfigurationBuilder builder, string? hostname = null, int? port = default, string? publicHostname = null, int? publicPort = default) { }
     }
-    public sealed class RemoteOptions
+    public sealed class DotNettyTcpTransportOptions : Akka.Remote.Hosting.DotNettyTransportOptions
+    {
+        public DotNettyTcpTransportOptions() { }
+    }
+    public abstract class DotNettyTransportOptions : Akka.Remote.Hosting.RemoteTransportOptions, Akka.Remote.Hosting.IWithHostNameAndPort, Akka.Remote.Hosting.IWithPublicHostNameAndPort
+    {
+        protected DotNettyTransportOptions() { }
+        public bool? EnableSsl { get; set; }
+        public string? HostName { get; set; }
+        public int? Port { get; set; }
+        public string? PublicHostName { get; set; }
+        public int? PublicPort { get; set; }
+        public Akka.Remote.Hosting.SslOptions Ssl { get; set; }
+        public override string TransportId { get; }
+        protected override void Build(System.Text.StringBuilder builder) { }
+    }
+    public sealed class DotNettyUdpTransportOptions : Akka.Remote.Hosting.DotNettyTransportOptions
+    {
+        public DotNettyUdpTransportOptions() { }
+    }
+    public interface IWithHostNameAndPort
+    {
+        string? HostName { get; set; }
+        int? Port { get; set; }
+    }
+    public interface IWithPublicHostNameAndPort : Akka.Remote.Hosting.IWithHostNameAndPort
+    {
+        string? PublicHostName { get; set; }
+        int? PublicPort { get; set; }
+    }
+    public class RemoteOptions
     {
         public RemoteOptions() { }
+        public System.Collections.Generic.List<Akka.Remote.Hosting.RemoteTransportOptions> EnabledTransports { get; set; }
```

Comment:
You know, I'm thinking that even though we could support this - it's probably a footgun for 99.9999999999999% of users. Make super-users use HOCON if they want this feature.